### PR TITLE
Small Flora Fixes

### PR DESCRIPTION
**Pull request by ZeeMajora**
Fixes many of the base game/dlc small flora to have proper scenery groups and the "isTree" flag.
Also fixes various of the expansion pack small flora objects having improper & restrictive flags applied to them.

The only ones i'm kind of uncertain of at the moment are the quarter tile trees & the giant garden flora.
The quarter tile trees are quite small and short but they are called trees so i added the "isTree" flag to them.
Some of the giant garden flora is quite big so i'm not sure if i should move them over to the trees group & add "isTree" to them.
For now i have put all of the giant garden flora in the shrubs group, I do feel like both of these may need some discussion.
Here are all the things i have currently fixed:

Base Game:
Jungle, Pagoda, Space, & Candy Trees added to trees scenery group & added the "isTree" flag added to them.
Jungle bushes, Jungle plants, & Giant garden flora added to shrubs & ornaments scenery group.
Jungle Flowers & Pagoda lily added to gardens scenery group.

DLC:
Panda Bamboo Shoots moved to trees scenery group & added the "isTree" flag to it.

Both Expansions:
Removed "isRotatable" flag from all small flora objects that had it with an exception for some of TT's swamp ferns which seem like they're meant to have manual rotations.

Wacky Worlds:
Removed the "hasPrimaryColour" & "isStackable" flags from all african/asian small flora.
Added "hasNoSupports" flag to asian bamboo bush.
Removed "isTree" flag from african low bush.
African low bush moved to the shrubs and ornaments scenery group.

Time Twister:
Removed the "requiresFlatSurface" flag from all small swamp flora & both alien trees.
Removed the "prohibitsWalls" flag from both alien trees.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
I would like to see these changes reverted:

- All additions to the Giant Garden theming
- Classification of Giant Garden and Candy trees as trees and adding them to those scenery groups.
- Classification of Bamboo Shots as a tree - I think it may count as as shrub, but not as a tree.

**Comment by ZeeMajora** [comment]
The swamp ferns in question are the ones from this set, with the exception of the one in the red tiles as it does seem like it should have random rotations as it doesn't have a deliberate shape to it like the others.
DeepinScreenshot_select-area_20220102191148
As for the others changes, I will work on them soon, are there any other changes that should be made? I was personally feeling a bit uncertain about classifying some of the small quarter tile trees as trees due to scenario backwards compatibility concerns.
Since it'd mean that any old scenario with the no tree removal restriction featuring these trees would have more land blocked than intended.

**Comment by ZeeMajora** [comment]
I have implemented all of the requested changes.

File: objects/rct2/scenery_small/rct2.scenery_small.tjp1.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
         "removalPrice": 13,
         "cursor": "CURSOR_TREE_DOWN",
         "height": 56,
+        "sceneryGroup": "rct2.scenery_group.scgshrub",
```
Comment (Gymnasiast) [inline objects/rct2/scenery_small/rct2.scenery_small.tjp1.json:13]: revert

File: objects/rct2/scenery_small/rct2.scenery_small.tjp2.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
         "removalPrice": 10,
         "cursor": "CURSOR_TREE_DOWN",
         "height": 36,
+        "sceneryGroup": "rct2.scenery_group.scgshrub",
```
Comment (Gymnasiast) [inline objects/rct2/scenery_small/rct2.scenery_small.tjp2.json:13]: revert

File: objects/rct2/scenery_small/rct2.scenery_small.tsp2.json [hunk 3]
```diff
@@ -10,6 +10,8 @@
         "removalPrice": 19,
         "cursor": "CURSOR_TREE_DOWN",
         "height": 80,
+        "sceneryGroup": "rct2.scenery_group.scgtrees",
```
Comment (Gymnasiast) [inline objects/rct2/scenery_small/rct2.scenery_small.tsp2.json:13]: making a note here, this is being discussed in the team chat.

File: objects/rct2/scenery_small/rct2.scenery_small.tjp1.json (hunk 1 above)
Comment (Gymnasiast) [reply objects/rct2/scenery_small/rct2.scenery_small.tjp1.json:13]:
```suggestion
```

File: objects/rct2/scenery_small/rct2.scenery_small.tjp2.json (hunk 2 above)
Comment (Gymnasiast) [reply objects/rct2/scenery_small/rct2.scenery_small.tjp2.json:13]:
```suggestion
```

**Pull request merged**